fastapi/dependencies/utils.py

```diff
@@ -247,36 +247,36 @@ def is_scalar_sequence_field(field: ModelField) -> bool:
 
 def get_typed_signature(call: Callable[..., Any]) -> inspect.Signature:
     signature = inspect.signature(call)
-    globalns = getattr(call, "__globals__", {})
+    is_true_function = (
```

Comment:
It might look a bit nicer here if you pass a tuple of arguments to isinstance.

```python
if isinstance(call, (FunctionType, MethodType, BuiltinFunctionType))
```

Reply:
Oh neat, thanks!